### PR TITLE
Update Enterprise Support description

### PR DESCRIPTION
**Pull request by bethadele**
Change the description of Enterprise Support to accurately reflect what we promise customers, per [this Slack conversation](https://gruntwork-io.slack.com/archives/C79R0C0SC/p1666379088111599).

### CODE REVIEW

File: pages/support/_enterprise-support.html [hunk 1]
```diff
@@ -4,11 +4,9 @@ <h2>Enterprise Support</h2>
     <p>
       Gruntwork also offers an enterprise support plan that includes:
       <ul>
-        <li>Live chat in phone/video calls</li>
-        <li>Design reviews</li>
-        <li>Code reviews</li>
-        <li>Prioritized bug fixes</li>
-        <li>Guaranteed same-day response times</li>
+        <li>SLA on response times</li>
```
Comment (ebeneliason) [inline pages/support/_enterprise-support.html:7]: We could still list our offerings here, e.g. "Next- or same-day response time SLA"

File: pages/support/_enterprise-support.html [hunk 2]
```diff
@@ -4,11 +4,9 @@ <h2>Enterprise Support</h2>
...
+        <li>Private shared Slack channel</li>
+        <li>Phone/video calls if we can’t resolve a problem async</li>
       </ul>
       For more information, please <a href="/contact">contact us</a>.
```
Comment (ebeneliason) [inline pages/support/_enterprise-support.html:11]: Should we refer them to our contract terms before suggesting that they contact us? Or do we specifically want to drive all potential enterprise conversations through the sales channel?

**Review by zackproser — APPROVED** [review]
Overall changes LGTM, so approving in case you're looking to make fast progress here.

**Comment by josh-padnick** [comment]
Thanks for putting this together, Beth. You know, it occurs to me that the sole difference between pro support and enterprise support is a faster SLA. There's really nothing else. 

Given that, I propose we take this:
image

and turn it into this:
image

Thoughts?

**Comment by oredavids** [comment]
Closing this given how long it has been open and not updated.

**Pull request closed without merging**